specs/phase1/beacon-chain.md

```diff
@@ -305,7 +297,7 @@ class ShardBlockWrapper(Container):
     shard_parent_root: Root
     beacon_parent_root: Root
     slot: Slot
-    body: ByteList[MAX_SHARD_BLOCK_CHUNKS * SHARD_BLOCK_CHUNK_SIZE]
+    body: List[byte, MAX_SHARD_BLOCK_CHUNKS * SHARD_BLOCK_CHUNK_SIZE]
```

Comment:
Why removal of helper type `ByteList`?

Reply:
Since phase0 doesn't use it, remerkleable doesn't have this type, and `List[byte, N]` will behave the same. Once we get into more serious phase 1 testing I can optimize it again.